test/e2e/ocp_alerts.go

```diff
@@ -55,7 +55,8 @@ var _ = Describe("Alerts", Label(smoke), Serial, func() {
 			Client: &http.Client{
 				Transport: config.NewAuthorizationCredentialsRoundTripper(
 					"Bearer",
-					config.Secret(token.Status.Token),
+					//config.Secret(token.Status.Token),
```

Comment:
nit: can we remove this comment before merge?

Reply:
good catch, forgot to remove this after testing, removed now.